recipes/default.rb

```diff
@@ -2,7 +2,6 @@
 # Cookbook:: foreman
 # Recipe:: default
 #
-include_recipe 'hostname'
 ohai 'reload' do
```

Comment:
If you're not setting the hostname you probably don't need to reload ohai

Reply:
Good point. i didnt think setting the hostname should be part of foreman so ill remove that too.